image-search/Models/Alert.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  Alert.swift
+//  image-search
+//
+//  Created by Renato Vieira on 22/10/19.
+//  Copyright © 2019 Stant. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class Alert: NSObject {
+
```

Comment:
Extra line

Reply:
This class has been removed and the function placed in the view controller class as suggested.